deps/rabbitmq_stream/src/rabbit_stream_reader.erl

```diff
@@ -2376,6 +2487,137 @@ handle_frame_post_auth(Transport,
+                                         [OffsetSpec]),
+                        QueueResource =
+                            #resource{name = Stream,
+                                      kind = queue,
+                                      virtual_host = VirtualHost},
+
+                        Segment =
+                            init_reader(ConnTransport,
+                                        LocalMemberPid,
+                                        QueueResource,
+                                        SubscriptionId,
+                                        Properties,
+                                        OffsetSpec),
+                        Consumer1 = Consumer#consumer{log = Segment},
+                        Consumer2 =
+                            case send_chunks(Transport, Consumer1, SendFileOct)
+                            of
+                                {error, closed} ->
+                                    rabbit_log_connection:info("Stream protocol connection has been closed by "
+                                                               "peer",
+                                                               []),
+                                    throw({stop, normal});
+                                {error, Reason} ->
+                                    rabbit_log_connection:info("Error while sending chunks: ~p",
+                                                               [Reason]),
+                                    %% likely a connection problem
+                                    Consumer;
+                                {ok, Csmr} ->
+                                    Csmr
+                            end,
+                        #consumer{configuration =
+                                      #consumer_configuration{counters =
+                                                                  ConsumerCounters},
+                                  log = Log2} =
+                            Consumer2,
+                        ConsumerOffset = osiris_log:next_offset(Log2),
+
+                        rabbit_log:debug("Subscription ~p is now at offset ~p with ~p message(s) "
+                                         "distributed after subscription",
+                                         [SubscriptionId, ConsumerOffset,
+                                          messages_consumed(ConsumerCounters)]),
+
+                        Consumers#{SubscriptionId => Consumer2};
+                    #{SubscriptionId :=
+                          #consumer{configuration =
+                                        #consumer_configuration{active = false,
+                                                                stream = Stream,
+                                                                properties =
+                                                                    Properties}}} ->
+                        rabbit_log:debug("Not an active consumer"),
```

Comment:
There are quite a lot of debug messages that won't make it into the final version I guess? These were just meant for development, right?